python/BioSimSpace/Sandpit/Exscientia/_SireWrappers/_molecule.py

```diff
@@ -502,6 +502,22 @@ def isDecoupled(self):
         else:
             return False
 
+    def isML(self):
+        """
+        Whether this molecule is marked as ML molecule, i.e. it can be used in a
+        MM to ML transformation simulation.
+
+        Returns
+        -------
+
+        is_ML : bool
+            Whether the molecule is marked as ML.
+        """
+        if self._sire_object.hasProperty("ML"):
```

Comment:
Is checking for `hasProperty` enough? Can this property not be set to `False` (e.g. from above `mol_edit.setProperty("ML", False)`) under circumstances?

Reply:
This shall be dealt with in the later stages.
This ML means a hybrid system, where lambda 0 is pure MM system and lambda 1 is a pure ML system.
A hybrid system at lambda 0, despite being a pure MM system will go down the same code route as lambda 1 pure ML system.